modules/javafx.graphics/src/test/java/test/com/sun/javafx/sg/prism/DirtyRegionTestBase.java

```diff
@@ -428,7 +418,6 @@ private void accumulateDirtyRegions() {
      */
     private void assertOnlyTheseNodesWereAskedToAccumulateDirtyRegions(NGNode start, Set<NGNode> nodes) {
         assertEquals(
-                "creator=" + creator + ", polluter=" + polluter,
```

Comment:
same comment here (and possibly elsewhere)
one thing you could do is to retain instance variables, just set them in the setup method - this should be safe, I don't think junit will invoke test methods for the same instance from multiple threads.